tests/unit/states/test_pdbedit.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Import Salt Libs
+import salt.states.pdbedit as pdbedit
+
+# Import Salt Testing Libs
+from tests.support.mixins import LoaderModuleMockMixin
+from tests.support.unit import TestCase
+from tests.support.mock import patch, MagicMock
+
+
+class PdbeditTestCase(TestCase, LoaderModuleMockMixin):
+    '''
+    TestCase for salt.states.pdbedit module
+    '''
+
+    def setup_loader_modules(self):
+        return {pdbedit: {}}
+
+    def test_generate_absent(self):
+        '''
+        Test salt.states.pdbedit.absent when
+        user is already absent
+        '''
+        name = 'testname'
+        with patch.object(pdbedit, '__salt__', return_value=MagicMock()):
```

Comment:
What does MagicMock() doe in this case? It sets __salt__ on pdbedit, but to what?

Reply:
i actually intended on mocking `pdbedit.list`  specificly, note the entire `__salt__` dict. this was a mistake on my part. thanks for pointing out. working on a fix.